doc/lua_api.txt

```diff
@@ -1304,7 +1304,7 @@ in the form of a table.  This table specifies the following fields:
   * `prob` (alias `param1`): the probability of this node being placed (default: 255)
   * `param2`: the raw param2 value of the node being placed onto the map (default: 0)
   * `force_place`: boolean representing if the node should forcibly overwrite any
-     previous contents (default: false)
+    previous contents (default: false)
```

Comment:
Lines 1303-1307 seem to need 2 more spaces of indent?

Reply:
Nice catch - will update

Reply:
Not sure, not essential.

Reply:
It should be done for consistency with other second level bullets, so I'd like to.

Reply:
Done.

Reply:
It's ok as it is.

Reply:
Now that I changed it it is. It was inconsistent.

Reply:
Sorry just saw your comments, the new PR is fine.